docs/packages/context-menu/context-menu-manager.md

```diff
@@ -267,79 +265,94 @@ The element triggers the `GlobalEventHandlers.oncontextmenu` when a right-click
 
 ```JS
 
-import { ContextMenuManager } from 'elementor/packages/context-menu/manager';
+import {
+	ContextMenuManager
+} from 'elementor/packages/context-menu/manager';
 
 // Element triggering the flow, when right-clicked on it:
 export const onContextMenu = () => {
 	const manager = new ContextMenuManager();
 
 	// Add groups that will be displayed first:
-	manager.createGroup(
-		{ id: 'groupId-1' }
-	);
+	manager.createGroup({
+		id: 'groupId-1'
+	});
 
 	manager.addItem(
-		'groupId-1',
-		{ id: 'some-item-id', label: 'Do some magic' }
+		'groupId-1', {
+			id: 'some-item-id',
+			label: 'Do some magic'
+		}
 	);
 
-	manager.createGroup(
-		{ id: 'groupId-2' }
-	);
+	manager.createGroup({
+		id: 'groupId-2'
+	});
 
 	manager.addItem(
-		'groupId-1',
-		{ id: 'another-item-id', label: 'Do another magic' }
+		'groupId-1', {
+			id: 'another-item-id',
+			label: 'Do another magic'
+		}
 	);
 
 	// Allow adding groups from Third party:
-	window.dispatchEvent( new CustomEvent( 'elementor/element/contextMenu', { detail: {
-			manager, 						// Send the manager instance
-			elType: 'widget',				// Send data about the triggerer
-			widgetType: 'awesome-widget3',	// Send data about the triggerer
-		} } ) );
+	window.dispatchEvent(new CustomEvent('elementor/element/contextMenu', {
```

Comment:
I would add some spaces here and there, just to comply with our coding style

Reply:
Done.